gateway/ln-gateway/src/lib.rs

```diff
@@ -914,7 +935,20 @@ impl Gateway {
         .await;
         dbtx.commit_tx_result()
             .await
-            .map_err(GatewayError::DatabaseError)
+            .map_err(GatewayError::DatabaseError)?;
+        if let GatewayState::Running { lnrpc, .. } = self.state.read().await.clone() {
+            let scids: Vec<u64> = self
+                .scid_to_federation
+                .read()
+                .await
+                .keys()
+                .cloned()
+                .collect();
+
+            lnrpc.update_fedimint_scids(scids).await?;
```

Comment:
Same comment as above, we shouldn't fail here if we fail to update the short channel ids.

Reply:
updated to warn and swallow